### ISSUE TITLE
AirPlay/Cast does not recognize Local Network Permission

### ISSUE THREAD

**Issue by Fannangir** [dayanch96/YTMusicUltimate#311]
### Describe the bug

```yaml
Pressing on the cast icon to select AirPlay receiver, the application asks for local network permission.
Turn the setting of Local Network u set settings On is not recognized by the application (following enable, it still asks to enable).
```

### Steps to reproduce

```yaml
1. Press AirPlay (Cast) icon.
2. Application asks for Local Network access.
3. Open Settings and enable Local Network.
4. Go back to YouTube Music, it asks for permission again when AirPlay (Cast) icon is pressed.
```

### Post Requisites

- [X] I have checked that a similar issue has not been reported before.
- [X] Verified the relevance of the project and the version you are using.
- [X] Provided a detailed description of the problem and have filled in all required fields.
- [ ] I have read the 'IPA Building Troubleshooting' section (Only for issues related to building the IPA file).
- [ ] I did not answer truthfully to **ALL** the above checkboxes.
- [X] By submitting this issue, **I confirm that I have filled in all the necessary fields and complied with the guidelines. Failure to comply may result in the issue being closed without discussion, and repeated violations may lead to restrict account from further participation.**

**Comment by Fannangir** [comment]
Fixed in YT 7.28

**Comment by Fannangir** [comment]
Unfortunately, cast does not work.
Is there a way to fix it?
I have tried to install the original app, allow permission, and install this tweaked ipa.
Unfortunately, cast screen does not show.
The screen does not show on local network only. It does appear when connected to 5G.

**Comment by dayanch96** [comment]
No, sideloading issues. There only ‘workarounds’ that I don’t want to mess with
